test/stdlib/DispatchDeprecationWatchOS.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %swift -typecheck -target i386-apple-watchos2.0 -verify -sdk %sdk %s
-// REQUIRES: OS=watchos
+// REQUIRES: CPU=x86_64, OS=watchos
```

Comment:
Thanks @gparker42, I was mostly thinking about the values of the tags that are used in Swift. They are all defined somewhere in `lit.cfg` but there is just no list with all of the possible tags and their values.